org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/consistencycheck/ConsistencyChecker.java

```diff
@@ -410,7 +418,12 @@ public void checkNamespaceUri(@NonNull DefinitionsChildId id) {
             })
             .anyMatch(definitionsChildName -> uriStr.contains(definitionsChildName));
         if (namespaceUriContainsDifferentType) {
-            printAndAddError(id, "Namespace URI contains tosca definitions name from other type. E.g., Namespace is ...servicetemplates..., but the type is an artifact template");
+            if ((id instanceof ServiceTemplateId) && (id.getNamespace().getDecoded().contains("compliance"))) {
```

Comment:
IMHO, this is not correct. Why do you even get the compliance rule as a `ServiceTemplateId`? The consistency checker should know the difference between a Service Template and a Compliance Rule.

Why is this `if` statement empty? If no action is required, flip the statement.

Reply:
This is the design by @ckrieger. And IMHO the idea of compliance rules: That hey are service templates by themselves.